recipes/coscine/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "coscine" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "058405f8835e45981e7f600978611355ba88ea265bdacefb362360af184aa07d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - requests
+    - requests-toolbelt
+    - tqdm
+    - colorama
+    - prettytable
```

Comment:
Those should only be `run` requirements. Pure python packages should only require `python` and `pip` in `host` requirements.
But I see that the package is importing itself in [setup.py](https://git.rwth-aachen.de/coscine/docs/public/coscine-python-sdk/-/blob/master/setup.py): `from src.coscine.version import __version__`.

This is not recommended:

> Although this technique is common, beware that it will fail if sample/__init__.py imports packages from install_requires dependencies, which will very likely not be installed yet when setup.py is run.

 Could you please open an issue in the upstream repo to change that in next version?
You can link to https://packaging.python.org/en/latest/guides/single-sourcing-package-version/


Recipe looks good otherwise. I'll merge.
Think about removing those run requirements when the upstream repo is updated.

Reply:
Thanks a lot for the comment and the merge, I will ask for a corresponding change for the next version.